OracleDatabase/README.md

```diff
@@ -12,6 +12,12 @@ To assist in building the images, you can use the [buildDockerImage.sh](dockerfi
 
 The `buildDockerImage.sh` script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call `docker build` with their prefered set of parameters.
 
+### Requirements for Building
+If you intend to build, make sure you have:
+
+1. Enough Docker storage space: 20 GB minimum
```

Comment:
It might be worth actually calling out how to verify this, particularly if the device-mapper storage driver is in use. Or at least link to the documentation.